src/main/java/com/amazonaws/services/kinesis/leases/impl/LeaseCoordinator.java

```diff
@@ -174,7 +173,7 @@ public LeaseCoordinator(ILeaseManager<T> leaseManager,
      * @param maxLeasesToStealAtOneTime Steal up to these many leases at a time (for load balancing)
      * @param metricsFactory Used to publish metrics about lease operations
      */
-    public LeaseCoordinator(ILeaseManager<T> leaseManager,
+    LeaseCoordinator(ILeaseManager<T> leaseManager,
```

Comment:
The `public` can't be removed. This class is a public class, this would be a breaking change.

Reply:
Made public.